src/peakWatcher.cpp

```diff
@@ -362,7 +362,7 @@ void Watcher::description(ostringstream& s) {
 	} else {
 		describeTarget(target, s);
 	}
-	s << " " << translate(this->levelTypeInfo().name);
+	s << " " << translate(this->levelTypeInfo().name) << " " << this->notifyLevel << " " << translate("threshold");
```

Comment:
This one probably needed code review. The problem with translating this way is that there's no context for the word "threshold", which might cause problems for some languages and certainly makes things harder for translators. For future reference, I probably would have done this:

```
s << " " << translate(this->levelTypeInfo().name) << " " <<
	// Translators: Used in the Peak Watcher menu to indicate the notification threshold for a watcher.
	// {} will be replaced with the level. For example: "0 threshold"
	format(translate("{} threshold"), this->notifyLevel);
```

There's probably no point doing this now since some translators have already translated it as is, but it's worth keeping in mind for future changes like this.